src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -101,22 +96,17 @@ public EventValueInfo(final Class<E> event, final Class<T> c, final Getter<T, E>
 	 * @param time
 	 * @param excludes Subclasses of the event for which this event value should not be registered for
 	 */
-	public static <T, E extends Event> void registerEventValue(final Class<E> e, final Class<T> c, final Getter<T, E> g, final int time, final @Nullable String excludeErrorMessage, final @Nullable Class<? extends E>... excludes) {
+	public static <T, E extends Event> void registerEventValue(Class<E> e, Class<T> c, Getter<T, E> g, int time, @Nullable String excludeErrorMessage, @SuppressWarnings("unchecked") @Nullable Class<? extends E>... excludes) {

```

Comment:
Don't just suppress heap pollution warnings. It would be better to manually verify that this occasion is safe and add @SafeVarargs to method.